script/foreman-admin

```diff
@@ -0,0 +1,181 @@
+#!/usr/bin/env ruby
+# -*- encoding: utf-8 -*-
+
+require 'rubygems'
+require 'clamp'
```

Comment:
Why would someone install Foreman without using the installer? Personally, I feel like that's on them to resolve dependencies if they don't use the installer.